applications/IgaApplication/tests/run_cpp_unit_tests.py

```diff
@@ -1,5 +1,6 @@
 from KratosMultiphysics import *
 from KratosMultiphysics.IgaApplication import *
+from KratosMultiphysics.StructuralMechanicsApplication import *
```

Comment:
why?

Reply:
For the constitutive laws. We do not have any here in the IGA application.

Reply:
IGA without the StructuralMechanicsApplication is not possible. Only for very few cases...

Reply:
Maybe we should add a dependency in the cmakelist to the StructuralMechanicsApp like it is done in other Apps?